diff --git a/dist/vite/vmod.js b/dist/vite/vmod.js
index 20dd46565f699144a3398059e7e52b5b266c5f30..ccaf7e8646ef167c77f71cfe3dab94c83bd7161b 100644
--- a/dist/vite/vmod.js
+++ b/dist/vite/vmod.js
@@ -13,8 +13,11 @@
 Object.defineProperty(exports, '__esModule', { value: true });
 
 let id = name => `virtual:remix/${name}`;
-let resolve = id => `\0${id}`;
-let url = id => `/@id/__x00__${id}`;
+let resolve = id => id;
+let url = id => id;
+// TODO: need to handle "\0" somewhere within vite-node path<->url conversion?
+// let resolve = id => `\0${id}`;
+// let url = id => `/@id/__x00__${id}`;
 
 exports.id = id;
 exports.resolve = resolve;
