diff --git a/dist/esm/index.js b/dist/esm/index.js
index b35fd97f752b2e3fe46a786f31eef4bc4aadb92a..1ba3579e8d9ef69f0dd073829be42c0a3f8e7afa 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -142,7 +142,7 @@ function createBrowserHistory(opts) {
     if (!scheduled) {
       rollbackLocation = currentLocation;
     }
-    currentLocation = parseHref(destHref, state);
+    currentLocation = parseHref(href, state);
     next = {
       href,
       state,
